### PR TITLE
feat: save events for delete booking and vehicle

### PR DESCRIPTION
**Pull request by Graborg**
This PR adds the events bookingDeleted and vehicleDeleted. 
It also enabled the engine-server to listen to rabbitmq messages concerning these events.

### CODE REVIEW

**Comment by Graborg** [comment]
[Add "Remove booking/vehicle" to eventstore](https://trello.com/c/Q3uhgIbg/265-add-remove-booking-vehicle-to-eventstore)

File: packages/engine-server/lib/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+const server = require('http').createServer()
+const io = require('socket.io')(server)
+require('events').defaultMaxListeners = 20
```
Comment (raduachim) [inline packages/engine-server/lib/index.js:3]: Why do we do this?

**Review by raduachim — APPROVED** [review]
It works but why do we increase the events max listeners to 20? 
Do we have an understanding where we keep adding the listeners?

**Comment by Graborg** [comment]
Yes, it seems fluent amqp uses new listeners for every subscription that we set up.

**Pull request merged**